portal.go

```diff
@@ -2143,11 +2135,6 @@ func (portal *Portal) HandleiMessage(msg *imessage.Message) id.EventID {
 	// If the message exists in the database, handle edits or retractions
 	if dbMessage != nil && dbMessage.MXID != "" {
 
-		// Clean up the text so the length is accurate
-		msg.Text = strings.ReplaceAll(msg.Text, "\ufffc", "")
-		msg.Subject = strings.ReplaceAll(msg.Subject, "\ufffc", "")
-
-		// DEVNOTE: It seems sometimes the message is just edited to remove data instead of actually retracting it
```

Comment:
I'd like to leave this dev note here for future reference.

Reply:
yes